### PR TITLE
application: nrf5340_audio: Refine DFU document

### PR DESCRIPTION
**Pull request by pirun**
Correct minor DFU document error.
Remove CONFIG_PARTITION_MANAGER_ENABLED, use overlay instead. Rename net core BLE controller.
Increase BT_BUF_ACL_RX_SIZE size for SMP in BIS mode.
Update release note.

Signed-off-by: Pirun Lee <pirun.lee@nordicsemi.no>

### CODE REVIEW

File: applications/nrf5340_audio/README.rst
Comment (alexsven) [inline applications/nrf5340_audio/README.rst:945]:
```suggestion
* Example 2: The following command builds both cores, using the DFU internal flash memory and the minimal size of the network core bootloader:
```

Comment (alexsven) [reply applications/nrf5340_audio/README.rst:945]: I thought we had to use external flash if both cores are updated?

**Review by alexsven — CHANGES_REQUESTED** [review]

File: applications/nrf5340_audio/README.rst
Comment (pirun) [reply applications/nrf5340_audio/README.rst:945]: We should build both core if DFU is enabled either internal or external layout.
Both cores can be updated one by one if internal flash.
Some of our customers might want to experience this feature without flash.

File: applications/nrf5340_audio/README.rst
Comment (rick1082) [inline applications/nrf5340_audio/README.rst:944]: Would "using internal flash memory for storing DFU image" be better?

**Review by rick1082 — APPROVED** [review]

**Review by erikrobstad — COMMENTED** [review]
I plan to review this PR tomorrow. Please to not merge before that.

File: applications/nrf5340_audio/README.rst
Comment (erikrobstad) [inline applications/nrf5340_audio/README.rst:1245]:
```suggestion
      The file for the network core follows the pattern ``dev_<Role>_build_<Application version>_net_core_app_update_<FW_VERSION>.bin``.
```

File: applications/nrf5340_audio/README.rst
Comment (erikrobstad) [inline applications/nrf5340_audio/README.rst:1246]: The example name for the application core should be moved to step a.

File: applications/nrf5340_audio/README.rst
Comment (erikrobstad) [inline applications/nrf5340_audio/README.rst:1248]: Maybe put this in a note?

``debug`` = ``build_debug``
``release`` = ``build_release``

Add "and" in stead of comma before ``<FW_VERSION>``

File: applications/nrf5340_audio/README.rst
Comment (erikrobstad) [inline applications/nrf5340_audio/README.rst:1257]:
```suggestion
#. Tap :guilabel:`START` and check :guilabel:`Confirm only` in the notification. Then tap :guilabel:`START` again to start the DFU process.
```

**Review by erikrobstad — CHANGES_REQUESTED** [review]

File: applications/nrf5340_audio/README.rst
Comment (pirun) [reply applications/nrf5340_audio/README.rst:948]: ``-c app`` also can create signed ble_ctrl.
I will revoke this change.

File: applications/nrf5340_audio/README.rst
Comment (pirun) [reply applications/nrf5340_audio/README.rst:944]: Revoke this change.

File: applications/nrf5340_audio/README.rst
Comment (pirun) [reply applications/nrf5340_audio/README.rst:969]: Revised.

File: applications/nrf5340_audio/README.rst
Comment (pirun) [reply applications/nrf5340_audio/README.rst:946]: Revoke this change.

File: applications/nrf5340_audio/README.rst
Comment (pirun) [reply applications/nrf5340_audio/README.rst:1244]: Revised.

File: applications/nrf5340_audio/README.rst
Comment (pirun) [reply applications/nrf5340_audio/README.rst:1246]: Multi-image DFU can use these files, zip, app core binary, net core.
Since we want multi-image to be DFU at once. We should choose zip file.
The app core/net core binary should go to single-image DFU description.

File: applications/nrf5340_audio/README.rst
Comment (pirun) [reply applications/nrf5340_audio/README.rst:1257]: Revised.

File: applications/nrf5340_audio/README.rst
Comment (alexsven) [reply applications/nrf5340_audio/README.rst]: Ok

**Review by alexsven — APPROVED** [review]

File: applications/nrf5340_audio/README.rst
Comment (greg-fer) [reply applications/nrf5340_audio/README.rst:1246]: I don't understand. Does this mean single-image uses 2 files and multi-image uses 1 file? That is totally counterintuitive :/

**Review by greg-fer — APPROVED** [review]
Approving, but please squash my PR before merge.

File: applications/nrf5340_audio/README.rst
Comment (erikrobstad) [inline applications/nrf5340_audio/README.rst:1006]: enable = enabled

File: applications/nrf5340_audio/README.rst
Comment (erikrobstad) [reply applications/nrf5340_audio/README.rst]: It does not look like this was related to your change. Can you try to remove one space so that the command line examples get tabbed the same? :)

File: applications/nrf5340_audio/README.rst
Comment (erikrobstad) [reply applications/nrf5340_audio/README.rst]: 2 .bin files are being generated for single core DFU (net.bin and app.bin), but only one file .bin file is used at a time

**Pull request merged**
